llama_index/embeddings/elasticsearch.py

```diff
@@ -0,0 +1,163 @@
+from typing import List, Any
+from llama_index.embeddings.base import BaseEmbedding
+
+from elasticsearch import Elasticsearch
+from elasticsearch.client import MlClient
```

Comment:
since elastic search isn't a core dependency, these imports will need to go in the init most likely (or any function it's used in), wrapped with an appropriate try/except